config/locales/es.yml

```diff
@@ -419,19 +419,19 @@ es:
         <p>La app fue creada para hacer todo el trabajo relacionado más fácil, tanto para los organizadores como las solicitantes potenciales. Los organizadores no tienen que crear formularios para las solicitudes de los tickets y pueden concentrarse en otro tipo de trabajo organizativo. Las solicitantes adquieren además una visión general de qué eventos están ofrenciendo diversity tickets a través de nuestra lista de eventos. </p>
         <p>Si estás interesada/o en proporcionar una experiencia en una conferencia más abierta para tus participantes, ¡%{email_diversitytickets} ponte en contacto con nosotras</a>!</p>
         <p>Travis Foundation ha colaborado con %{ruby_monstas} para crear esta aplicación de Código Abierto y Libre. Lee más sobre la idea y el concepto en el blog de Travis Foundation: %{travis_blog} %{app_to_diversify}</a>.</p>"
-      app_to_diversify: "An app to diversify tech conferences [EN]"
+      app_to_diversify: "Una app para mejorar la diversidad"
```

Comment:
I would keep that one as was before because the linked page is actually only available in english. 
Or add some hint that makes it clear the linked article is only in english.